Sources/TestingMacros/Support/AttributeDiscovery.swift

```diff
@@ -193,6 +180,22 @@ struct AttributeInfo {
         ArrayElementSyntax(expression: traitExpr)
       }
     }))
+
+    var otherArguments = self.otherArguments
+
```

Comment:
Stray blank line? Might be more concise to move the `var otherArguments` decl under the comment.